index.html

```diff
@@ -112,36 +113,41 @@ <h3>Meeting Minutes</h3>
 <section id="past-committee">
   <h2>Past Committee Members</h2>
   <ul>
+    <li>Ben Tillman (president 2022-2023)</li>
+    <li>Michael Yin (secretary 2021-2023)</li>
+    <li>Mark Haylock (general member 2021-2023)</li>
+    <li>Sarah Hay (general member 2022-2023)</li>
+    <li>Matthew Wratt (general member 2022-2023</li>
```

Comment:
This is missing a closing bracket after 2023

Reply:
Yes, Michael, Mark, Sarah, Mathew carried on to 2024, Only Ben handed over Presidency to Sarah.